src/sql/media/icons/common-icons.css

```diff
@@ -33,6 +33,19 @@
 	background: url("database_inverse.svg") center center no-repeat;
 }
 
+.vs-dark .codicon.icon-dashboard-view,
+.hc-black .codicon.icon-dashboard-view,
+.vs .codicon.icon-dashboard-view {
+	background: url("dashboard_view.svg") center center no-repeat;
+}
+
+.vs-dark .codicon.icon-dashboard-view::before,
+.hc-black .codicon.icon-dashboard-view::before,
+.vs .codicon.icon-dashboard-view::before {
+	-webkit-mask-image: url("dashboard_view.svg");
+	mask-image: url("dashboard_view.svg");
+}
```

Comment:
is this really needed? the same image is being used twice once as a background and then as a mask image of the before modifier?

Reply:
You were right, that wasn't needed. I removed it.